src/themes/irokoto-forts/collections/form.overrides

```diff
@@ -281,6 +281,14 @@
         z-index: 1;
     }
 
+    .st_radio_report:after {
```

Comment:
Please use CSS3 syntax `::` instead of `:`.
https://developer.mozilla.org/en-US/docs/Web/CSS/::before